### PR TITLE
Culture page

### PR DESCRIPTION
**Pull request by dean-mediasuite-nz**
December Culture page changes:

- removed freedom and accountability, people and community, learning and growth, value and integrity

- added mindset over skillset, trust our people, teaching and learning, and unashamed geeks.

### CODE REVIEW

**Comment by jooleearr** [comment]
@tfentonz I think I told him to just comment it while he was working on this stuff as I need to pull it down and test that removing it won't break anything. Will do so now.

But yep, agree that commented code shouldn't be committed normally. Took me a while to trust in git when it came to that :)

File: src/css/style.css [hunk 1]
```diff
@@ -490,8 +511,8 @@ a {
 }
 
 #page-header.culture-header:before {
-  content: none;
-}
+/*  content: none;
+*/}
```
Comment (jooleearr) [inline src/css/style.css]: Best to remove this entire declaration if it's no longer needed 👍

File: src/css/style.css [hunk 2]
```diff
@@ -892,9 +913,17 @@ a {
...
+  width: 100%;
+  margin: 0 0 30px 0;
+}
+
+@media (min-width: 768px) {
+  
 .col-half {
   width: 45%;
   margin: 0 2%;
+}
```
Comment (jooleearr) [inline src/css/style.css]: Please fix the indentation here :)

**Comment by dean-mediasuite-nz** [comment]
Fixed those issues from December: tidied CSS, renamed helper classes to be more descriptive, removed empty css rules.

**Pull request merged**